lib/home_screen/home_screen.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:lachies_life_planner/home_screen/widgets/text_icon_button.dart';
+import 'package:lachies_life_planner/shared/size_config.dart';
+
+class HomeScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    initSizeConfigurations(MediaQuery.of(context).size);
+
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Home'),
+      ),
+      body: Container(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            _ExpandedRow(
+              children: <Widget>[
+                Expanded(
+                  child: TextIconButton(
+                    onPressed: null,
+                    text: 'Tasks',
+                    icon: Icons.check,
+                  ),
+                ),
+                Expanded(
+                  child: TextIconButton(
+                    onPressed: null,
+                    text: 'Calendar',
+                    icon: Icons.calendar_today,
+                  ),
+                ),
+              ],
+            ),
```

Comment:
Can this be simplified?  Maybe `_ExpandedRow` could also expand it's children?

Or could a GridView work?

Reply:
Nah this is better